delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -204,19 +215,21 @@ class AclsRoutes(identities: Identities, acls: Acls)(implicit
             }
           },
           (get & extractAclAddressFilter) { addressFilter =>
-            parameter("self" ? true) {
-              case true  =>
-                completeSearch(
-                  acls
-                    .listSelf(addressFilter)
-                    .map(aclCol => SearchResults(aclCol.value.size.toLong, aclCol.value.values.toSeq))
-                )
-              case false =>
-                completeSearch(
-                  acls
-                    .list(addressFilter)
-                    .map(aclCol => SearchResults(aclCol.value.size.toLong, aclCol.value.values.toSeq))
-                )
+            operationName(s"$prefixSegment/acls/list") {
+              parameter("self" ? true) {
+                case true  =>
+                  completeSearch(
+                    acls
+                      .listSelf(addressFilter)
+                      .map(aclCol => SearchResults(aclCol.value.size.toLong, aclCol.value.values.toSeq))
+                  )
+                case false =>
+                  completeSearch(
+                    acls
```

Comment:
we would need authorization here as well, wouldn't we?

Reply:
Actually, we need filtering here. Current behaviour is that when user list e.g. `/*/*` we show him all the ACLs that match the pattern to which user has access.